mscore/pluginManager.cpp

```diff
@@ -49,14 +50,37 @@ void PluginManager::init()
             localShortcuts[s->key()] = new Shortcut(*s);
       shortcutsChanged = false;
 
-      preferences.updatePluginList();
-      int n = preferences.pluginList.size();
+      loadList(false);
+      connect(pluginList, SIGNAL(itemChanged(QListWidgetItem*)), SLOT(pluginLoadToggled(QListWidgetItem*)));
+      connect(pluginList, SIGNAL(currentItemChanged(QListWidgetItem*, QListWidgetItem*)),
+         SLOT(pluginListItemChanged(QListWidgetItem*, QListWidgetItem*)));
+      }
+
+//---------------------------------------------------------
+//   loadList - populate the listbox.
+//---------------------------------------------------------
+
+void PluginManager::loadList(bool forceRefresh)
+      {
+      QStringList saveLoaded; // If forcing a refresh, the load flags are lost. Keep a copy and reapply.
+      int n=preferences.pluginList.size();
+      if (forceRefresh && n>0) {
+            for(int i=0; i<n; i++) {
```

Comment:
space between `for`and `(`, spaces before and after `=``and `>`

Reply:
Has anyone told you you're picky?

Reply:
I am by nature ;-)

Reply:
Ready for more pickiness? There are more spaces missing, around `+` and `=` later in your PR